controllers/cruisecontroltask_controller.go

```diff
@@ -196,7 +196,7 @@ func (r *CruiseControlTaskReconciler) handlePodAddCCTask(kafkaCluster *v1beta1.K
 	cc := scale.NewCruiseControlScaler(kafkaCluster.Namespace, kafkaCluster.Spec.GetKubernetesClusterDomain(), kafkaCluster.Spec.CruiseControlConfig.CruiseControlEndpoint, kafkaCluster.Name)
 	uTaskId, taskStartTime, scaleErr := cc.UpScaleCluster(brokerIds)
 	if scaleErr != nil {
-		log.Info("cruise control communication error during upscaling broker(s)", "brokerId(s)", brokerIds)
+		log.Info("Cannot upscale broker(s)", "brokerId(s)", brokerIds, "error", scaleErr.Error())
```

Comment:
updated the wording here - as this can fail not only when there is a communication error with but also when say the broker is not yet ready and not yet registered in cc.
logging the error message as well for better clarity.